src/WebApp/Pages/Shared/_Header.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@using Microsoft.AspNetCore.Mvc.Localization
+@inject IViewLocalizer LayoutLocalizer
+<header class="header">
+    <div class="container header--container">
+        <div class="header--brand">
+            <a class="header--site-name" asp-area="" asp-page="/Index">Codidact</a>
+        </div>
+        <div class="header--menu">
+            <a class="header--item" href="#">@LayoutLocalizer["Questions_title"]</a>
+            <a class="header--item" href="#">@LayoutLocalizer["Tags_title"]</a>
```

Comment:
These two entries will be category-dependent and should be moved into a category header (see comment on CategoryHeader.cshtml)